src/admin/content/views/ContentEdit.tsx

```diff
@@ -145,7 +145,7 @@ const ContentEdit: FunctionComponent<ContentEditProps> = ({ history, match, user
 			title: contentForm.title,
 			description: contentForm.description || null,
 			is_protected: contentForm.isProtected,
-			path: contentForm.path,
+			path: contentForm.path || null,
```

Comment:
Moet ook nog aangepast worden in typings dan, die gaat nu niet klagen omdat hierboven nog `| any` staat

Reply:
https://github.com/viaacode/avo2-typings/pull/44